frontend/packages/dev-console/src/components/helm/HelmReleaseDetailsPage.tsx

```diff
@@ -52,7 +54,16 @@ export const LoadedHelmReleaseDetailsPage: React.FC<LoadedHelmReleaseDetailsPage
   if (_.isEmpty(secretResource)) return <ErrorPage404 />;
 
   const secretName = secretResource[0]?.metadata.name;
-  const releaseName = secretResource[0]?.metadata.labels?.name;
+  const releaseName = helmReleaseData?.name;
+
+  const title = (
+    <>
+      {releaseName}
+      <Badge isRead style={{ verticalAlign: 'middle', marginLeft: 'var(--pf-global--spacer--md)' }}>
```

Comment:
nit: can we have these styles in class

Reply:
I really don't like creating `scss` files and classes just for a few styles that are not going to be used anywhere else in the component. These are very specific to just the Badge component here.